### PR TITLE
Something idk

### PR DESCRIPTION
**Pull request by HurricanKai**
1. PUSH BEFORE YOU LEAVE
2. pls fix naming.

i added some random amount logic in the, i know, wrong command (the other isnt pushed)
i also added the "no 🤖 "

### CODE REVIEW

**Comment by HurricanKai** [comment]
btw, dont merge, just here to tell im wanting to help while not at pc, and well prob. add more shit in future

**Comment by Bentechy66** [comment]
Some of this isn't going to work, you have incorrect syntax in some places.

File: main.py [hunk 1]
```diff
@@ -15,8 +15,12 @@
 @client.event
 async def on_message(message):
     # we do not want the bot to reply to itself
+    # technically useless cause self is a bot too, tought fetching .bot may be more expensive?
     if message.author == client.user:
```
Comment (Bentechy66) [inline main.py]: No point also checking if the client is identical if you're already checking for a bot

File: main.py [hunk 2]
```diff
@@ -42,13 +46,17 @@
     # ----------------------------------------
     #               MARKET
     # ----------------------------------------
-    if message.content.startswith('!m'):
+    elif message.content.startswith('!m'):
```
Comment (Bentechy66) [inline main.py]: Whilst this is fine, it is slightly inefficient as you `return` out of the above if statement anyway, so there isn't any point wasting processing time checking if it failed or not

File: main.py [hunk 3]
```diff
@@ -42,13 +46,17 @@
...
-        if len(message.content.split(' ')) > 1:
-
-            emoji = message.content.split(' ')[1]
-
+        args = message.content.split(' ')
+        if len(args) > 1: #shoudnt this be 2? or does len actually return count?
+
+            emoji = args[1]
+            amountS = args[2]
+            if (!amountS)
```
Comment (Bentechy66) [inline main.py]: Incorrect `if` syntax for Python

**Review by Bentechy66 — CHANGES_REQUESTED** [review]

File: main.py (hunk 3 above)
Comment (HurricanKai) [reply main.py]: Oh right...

File: main.py (hunk 2 above)
Comment (HurricanKai) [reply main.py]: oh, didnt know this is the way in Python, in C#, this woud be more efficent, and

**Comment by HurricanKai** [comment]
better

File: main.py [hunk 4]
```diff
@@ -46,15 +43,15 @@
...
+    if message.content.startswith('!m'):
 
         # When a specific emoji has been chosen.
         args = message.content.split(' ')
         if len(args) > 1: #shoudnt this be 2? or does len actually return count?
 
             emoji = args[1]
             amountS = args[2]
-            if (!amountS)
+            if !amountS
```
Comment (Bentechy66) [inline main.py]: This requires a colon at the end

**Review by Bentechy66 — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (Randium) [reply main.py]: It doesn't seem to notice if I leave that piece of code out.

File: main.py (hunk 4 above)
Comment (ed588) [reply main.py]: It should be `not` instead of `!` as well

**Comment by Randium** [comment]
Hrm?

**Pull request closed without merging**